java/java-impl/src/META-INF/JavaPlugin.xml

```diff
@@ -1373,7 +1373,7 @@
     <localInspection language="JAVA" shortName="ConditionalBreakInInfiniteLoop" enabledByDefault="true" level="WARNING"
                      groupPath="Java" groupBundle="messages.InspectionsBundle" groupKey="group.names.control.flow.issues"
                      bundle="messages.JavaBundle" key="inspection.conditional.break.in.infinite.loop.description"
-                     implementationClass="com.intellij.codeInspection.ConditionalBreakInInfiniteLoopInspection"/>
+                     implementationClass="com.intellij.codeInspection.ConditionalBreakInInfiniteLoopInspection" cleanupTool="true"/>
```

Comment:
I would avoid making inspection cleanup, some people may prefer other code style, especially when it puts several expression in a single line of code. In such a case it will be more complicated to debug it.

Reply:
Done